terraform_test_artifacts/terraform_install.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -e
+
+USAGE="${0}: [<version>] 
+Where:
+  <version>: version of to dowload. If absent, it will download the latest.
+"
+
+DOWNLOAD_URL=
+if [ $# -eq 0 ] ; then
+	# lookup the latest release download URL
+	DOWNLOAD_URL="https://releases.hashicorp.com/terraform/1.0.9/terraform_1.0.9_linux_amd64.zip"
+	VER="1.0.9"
```

Comment:
Instead of hardcoding the version, you can get it using github API. For example:

`curl -s https://api.github.com/repos/hashicorp/terraform/releases/latest | grep '"name":' | awk '{print $2}' | sed 's/.*v//' | sed 's/".*//'`